rrrspec-client/lib/rrrspec/client/support.rb

```diff
@@ -40,7 +40,26 @@ def start_taskset(conf, rsync_name)
         return taskset
       end
 
-      def show_result(taskset, verbose=false)
+      def show_result(taskset, verbose=false, show_errors=false)
+        if show_errors
+          puts "Errors:"
+          taskset.tasks.each do |task|
+            if task.status == "failed"
+              trial = task.trials.first
```

Comment:
I think the first trial doesn't necessarily the failed trial. (e.g. `'timeout'`, `'error'`)